include/rppdefs.h

```diff
@@ -1136,4 +1136,18 @@ typedef struct
     memMgmt mem;
 } InitHandle;
 
+// sets descriptor dimensions and strides of fog mask
+inline void set_fog_mask_descriptor(RpptDescPtr descPtr, int batchSize, int maxHeight, int maxWidth, int numChannels)
```

Comment:
Is this the first time we have an inline helper in rppdefs.h? If there are more, suggest adding them in a header -  include/rppt_api_helpers.h and include it in rppt_tensor_effects_augmentations.cpp
We need doxygen style documentation ideally for anything that goes in external headers

Reply:
Moved that helper function to rppt_api_helpers.h file .